specification/columns/skuid.md

```diff
@@ -1,13 +1,38 @@
+
+* A product license that is purchased or subscribed to.
+* Usage of a deployed resource from direct user interaction (e.g., request count).
+* Usage by a deployed resource based on the resource's configuration (e.g., running hours, storage space).
+
+Each SKU ID represents a unique set of features can be sold at different price points or [*SKU Prices*](#glossary:sku-price). SKU ID is consistent across all pricing variations, which may differ based on multiple factors beyond the common functionality and technical specifications. Examples include but are not limited to:
+
+* Date the charge was incurred.
+* Provider agreements and contracts.
+* Pricing models (e.g., flat rate, tiered, committed).
+* Geographic region.
+* Service Level Agreements (SLAs).
+* Availability (e.g., spot VMs).
+
+SKU ID should be consistent across pricing variations of a good or service to facilitate price comparisons for the same functionality, like where the functionality is provided or how it's paid for. SKU ID can be referenced on a catalog or [*price list*](#glossary:price-list) published by a provider to look up detailed information about the *SKU*. The composition of the properties associated with the SKU ID may differ across providers. SKU ID is commonly used for analyzing and comparing costs for the same SKU in different regions, tiers, and more.
 
 The SkuId column adheres to the following requirements:
 
-* The SkuId column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider publishes a SKU list.
-* This column MUST be of type String.
-* It MUST NOT be null when [ChargeClass](#chargeclass) is not "Correction" and [ChargeCategory](#chargecategory) is "Usage" or "Purchase", MUST be null when ChargeCategory is "Tax", and MAY be null for all other combinations of ChargeClass and ChargeCategory.
-* SkuId MUST equal SkuPriceId when a provider does not support an overarching SKU ID construct.
+* SkuId MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider supports unit pricing concepts and publishes price lists, publicly or as part of contracting.
+* SkuId MUST be of type String.
+* SkuId MUST conform to [String Handling](#stringhandling) requirements.
+* SkuId nullability is defined as follows:
+  * SkuId MUST be null when [ChargeCategory](#chargecategory) is "Tax".
+  * SkuId MUST NOT be null when ChargeCategory is "Usage" or "Purchase" and [ChargeClass](#chargeclass) is not "Correction".
+  * SkuId MAY be null in all other cases.
+* SkuId MUST equal the SkuId for all other charges for the same *SKU* regardless of factors that impact the price but not the functionality.
+  * SkuId MUST be consistent across billing accounts or agreements.
+  * SkuId MUST be consistent across [PricingCategory](#pricingcategory) values..
+  * SkuId MUST be consistent across regions.
+  * SkuId MUST be consistent across service level agreement (SLA) variations..
+  * SkuId MUST be consistent across availability options.
```

Comment:
I debated whether to use a one-liner or break this out into separate requirements that are more independently enforceable. I felt separate is better because it allows conformance to be more precise, but I'm happy to go either way. Here's the alternative:
```suggestion
* SkuId MUST equal the SkuId for all other charges for the same *SKU* regardless of billing account type or agreement, [PricingCategory](#pricingcategory), region, SLA, availability, or other factors that impact the price but not the functionality.
```